packages/client/uno.config.ts

```diff
@@ -13,6 +13,8 @@ export default defineConfig({
   safelist: [
     '!opacity-0',
     'prose',
+    'grid-rows-[1fr_max-content]',
```

Comment:
We could add a comment above, so we know why we have them and could remove it in the future if we figured out why